src/main/java/com/volunteer/enterprise/dto/Volunteer.java

```diff
@@ -9,5 +9,13 @@
     private String volunteerUser;
     private String volunteerPass;
     public String testField;
+
+    public String toString() {
+        return "Volunteer - " +
+                " ID: " + volunteerID +
+                " Email: " + volunteerEmail +
+                " User: " + volunteerUser +
+                " Password: " + volunteerPass;
+    }
```

Comment:
Implemented more readable toString() than what Lombok provides for `Company` and `Volunteer` DTOs.